master/static/migrations/20231208132223_drop-raw-checkpoints.tx.up.sql

```diff
@@ -0,0 +1,16 @@
+DROP TABLE raw_checkpoints CASCADE;
+
+ALTER TABLE public.checkpoints_v2
+    ALTER COLUMN id SET NOT NULL;
```

Comment:
I could add a `DEFAULT` value here, but ids were previously assigned based on the `nextval` of a sequence created for `raw_checkpoints` which had a `NOT NULL` constraint, so previously inserted `checkpoints_v2` rows should never have an empty id column.